Initial Work

paulaOliverosHollis opened the pull request:
GameManager Class - was created to handle all the logistics of playig the game. Right now it simply prints the intro of the game.

TimedEvents - right now it has only one method which pritns a string char by char at a specified speed.

paulaOliverosHollis committed:
GameManager Class - was created to handle all the logistics of playing the game.

TimedEvents - right now it has only one method which pritns a string char by char at a specified speed.

zhollis21 commented on src/LostAndFound/TimedEvents.cs:
```diff
@@ -0,0 +1,19 @@
+{
+    public class TimedEvents
+    {
+        public enum WriteSpeed { Normal = 40, Fast = 20, Slow = 80 }
```
Great idea!

zhollis21 commented on src/LostAndFound/GameManager.cs:
```diff
@@ -0,0 +1,19 @@
+
+        private static void PrintIntro()
+        {
+            TimedEvents.TimedWrite(
+                "\n\n\t\tLost and Found..." +
+                "\n\t\t\tThe Ultimate Survival Game..." +
+                "\n\n\t\t\tWritten and Developed by: Zach Hollis\n\n",
+                TimedEvents.WriteSpeed.Slow);
```
Personally, I find it hard to tell the difference where we are passing separate args and where we are doing string concatenation. So I would recommend building the string separately from the function call.

```suggestion
            var introText = 
                "\n\n\t\tLost and Found..." +
                "\n\t\t\tThe Ultimate Survival Game..." +
                "\n\n\t\t\tWritten and Developed by: Zach Hollis\n\n";
                
            TimedEvents.TimedWrite(introText, TimedEvents.WriteSpeed.Slow);
```

zhollis21 replied:
Also we need to add your name to the intro credits. We should talk about what we want it to say.

zhollis21 commented on src/LostAndFound/TimedEvents.cs:
```diff
@@ -0,0 +1,19 @@
+
+        public static void TimedWrite(string text, WriteSpeed speed = WriteSpeed.Normal)
+        {
+            foreach (char letter in text)
```
This will cause an exception if null is passed. We should guard against that.
![image](https://user-images.githubusercontent.com/43797367/85306941-33220280-b47d-11ea-99b0-e31673d1f913.png)

paulaOliverosHollis replied:
What do you suggest?

paulaOliverosHollis committed:
Update src/LostAndFound/GameManager.cs

Built intro string separate from the print function.

Co-authored-by: Zach Hollis <43797367+zhollis21@users.noreply.github.com>

paulaOliverosHollis committed:
Resolved minor issues.
GameManager - moved the intro text into its own variable instead of adding it directly in the funtion call.
TimedEvents - added a null or empty string check to the TimedWrite method in order to avoid possible unhandled exceptions.

zhollis21 reviewed (approved):
Nice work!
